src/java/picard/analysis/directed/CalculateHsMetrics.java

```diff
@@ -45,16 +45,54 @@
  *
  * @author Tim Fennell
  */
+
 @CommandLineProgramProperties(
-        usage = "Calculates a set of Hybrid Selection specific metrics from an aligned SAM" +
-                "or BAM file. If a reference sequence is provided, AT/GC dropout metrics will " +
-                "be calculated, and the PER_TARGET_COVERAGE option can be used to output GC and " +
-                "mean coverage information for every target.",
-        usageShort = "Calculates Hybrid Selection-specific metrics for a SAM or BAM file",
+        usage = CalculateHsMetrics.USAGE_SUMMARY + CalculateHsMetrics.USAGE_DETAILS,
+        usageShort = CalculateHsMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CalculateHsMetrics extends CollectTargetedMetrics<HsMetrics, HsMetricCollector> {
+    static final String USAGE_SUMMARY = "Calculates hybrid Selection-specific metrics from aligned SAM or BAM files.  ";
+    static final String USAGE_DETAILS = "Hybrid selection (HS) enables targeted sequencing analysis via the capture of specified genomic DNA " +
+            "sequences (doi:10.1038/nbt.1523).  It is commonly used to characterize exon sequences from genomic DNA " +
+            "or filter out bacterial DNA sequences from clinical samples.  <br /><br /> " +
+            "" +
+            "The technique involves the capture of unique regions of genomic DNA (targets) using synthetic RNA (or DNA) baits." +
+            "   The baits are synthesized with biotinylated nucleotides to facilitate capture of bait:target hybrids on" +
+            " streptavidin beads.  The captured target sequences are amplified, sequenced, and processed for variant calling <br /><br />." +
+            "" +
+            "This tool requires an aligned SAM or BAM file as well as bait and target interval files.  " +
+            "The bait and target files are VCF formatted.  For information on VCF files, please see:" +
+            "<br /><br />www.1000genomes.org/wiki/analysis/variant%20call%20format/vcf-variant-call-format-version-41" +
+            "" +
+            "<br /><br />Tool provides multiple outputs including:" +
+            "<li>The numbers of aligned reads that pass the sequencing vendor's quality filter (PF) " +
+            "<li>The numbers of reads and their constituent bases that map to the reference sequence " +
+            "<li>The per read and per base coverage depth for both bait and target intervals<br /><br />" +
+            "The tool also calculates hybrid selection penalty values; defined as the cost incurred to get 80% of target bases" +
```

Comment:
Actually it is a list with bullet points.  I believe (might be wrong) that a colon should precede a list.